src/OpenTelemetry.SemanticConventions/scripts/generate.ps1

```diff
@@ -2,8 +2,8 @@ $SCRIPT_DIR = $PSScriptRoot
 $ROOT_DIR = "${SCRIPT_DIR}/../"
 
 # freeze the spec version to make SemanticAttributes generation reproducible
-$SPEC_VERSION = "1.25.0"
-$GENERATOR_VERSION = "latest"
+$SEMCONV_VERSION = "1.26.0"
```

Comment:
I believe weaver won't work properly with 1.25.0, so this is necessary.

Nit: could you please call semconv version update out in the PR title/description?

Reply:
Done!